manifests/hub/manifestworks.crd.yaml

```diff
@@ -0,0 +1,182 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
This CRD should be copied into the image from the vendored open-cluster-management/api repo

Reply:
Strong +1 on this.

Reply:
I think manifestwork/spokecluster crd is installed only when HubCore API is created on hub cluster? Currently I generate bindata with these crds. I will update the makefile, so it will generate bindata from vendor directory.

For nucleus CRD, agree we should copy into image from vendored api repo